tests/lib/assertions/valid-for-testing-pc-24.model

```diff
@@ -0,0 +1,44 @@
+type: model
+authority-id: test-snapd
+series: 16
+brand-id: test-snapd
+model: my-model
+architecture: amd64
+base: core24
+grade: dangerous
+serial-authority:
+  - generic
+snaps:
+  -
+    default-channel: 24/edge
+    id: UqFziVZDHLSyO3TqSWgNBoAdHbLI4dAH
+    name: pc
+    type: gadget
+  -
+    default-channel: 24/edge
```

Comment:
We need to change the channel for the kernel in the assertions from `24/edge` to `24/beta`, as edge kernels are not signed with the canonical key anymore.